src/screens/Menu.js

```diff
@@ -373,15 +359,6 @@ const MenuPage = () => (
           </PropertyValue>
         </Property>
 
-        <Property name="menu.group.separator.size">
-          <Description>
-            The thickness of the separator line between grouped items.
-          </Description>
-          <PropertyValue defaultValue type="string">
-            <Example>"xsmall"</Example>
-          </PropertyValue>
-        </Property>
-
```

Comment:
Why was this removed?

Reply:
opps mistake Ill put back

Reply:
done I just reordered them to be alphabet